content/en/blog/posts/2021-12-17-build-release-v0.7.0.md

```diff
@@ -0,0 +1,124 @@
+kubectl apply -f https://github.com/shipwright-io/build/releases/download/v0.7.0/release.yaml
+kubectl apply -f https://github.com/shipwright-io/build/releases/download/v0.7.0/sample-strategies.yaml
+```
+
+### shp Command Line
+
+For the CLI, we still have no package manager support. :-( You therefore need to get the release again from GitHub:
+
+#### Windows
+
+```sh
+$ curl --silent --fail --location https://github.com/shipwright-io/cli/releases/download/v0.7.0/cli_0.7.0_windows_x86_64.tar.gz | tar xzf - shp.exe
+$ shp help
+```
+
+#### Mac
+
+```sh
+$ curl --silent --fail --location https://github.com/shipwright-io/cli/releases/download/v0.7.0/cli_0.7.0_macOS_x86_64.tar.gz | tar -xzf - -C /usr/local/bin shp
+$ shp help
+```
+
+#### Linux
+
+```sh
+$ curl --silent --fail --location "https://github.com/shipwright-io/cli/releases/download/v0.7.0/cli_0.7.0_linux_$(uname -m | sed 's/aarch64/arm64/').tar.gz" | sudo tar -xzf - -C /usr/bin shp
+$ shp help
+```
+
+### Operator
+
+Coming soon! We are in the process of adding the new version of the Shipwright operator to [OperatorHub](https://operatorhub.io).
+Once the new version has been added, you can follow the provided instructions on OperatorHub to install the operator with OLM.
+If you previously installed v0.1.0 of the Shipwright operator, you must remove it first.
+
+Once the operator has been installed, you will be able to deploy the Build controller and APIs by creating an instance of the `ShipwrightBuild` custom resource:
+
+```yaml
+apiVersion: operator.shipwright.io/v1alpha1
+kind: ShipwrightBuild
+metadata:
+  name: shipwright-build
+spec:
+  targetNamespace: shipwright-build
+```
+
+## What's next ?
+
+We have cool stuff being under development. So, let me list three items that are all cool and are in the makings.
+
+Soon you should be able to see an evolution of parameters: build strategy authors will be able to define array parameters which is required for scenarios such as build-args for Dockerfile-based builds, and Build users will be able to define that the value should be retrieved from a ConfigMap or Secret.
+
+We work on the next evolution of error reporting inside BuildRuns. Where you today just see a BuildRun being failed and must look at the Pod logs to determine the root cause, you will see further error reasons and messages in the BuildRun status directly – with those details provided internally in our Shipwright-managed steps (like the Git step, for example when a revision does not exist), but also by the build strategy author (for example if Buildpacks fail to detect any source code, or a Dockerfile-based strategy when the Dockerfile does not exist).
+
+Finally, we investigated and spiked on different approaches to support building from source code from your local disk rather than from a Git repository. Part of the code is even in the product already, but we need more time to get it done end-to-end.
```

Comment:
ok I would definitely mention build volumes here :-)

Reply:
done in the preceding paragraph